### PR TITLE
Improve documentation of homeworks.send_command

### PR DESCRIPTION
**Pull request by emontnemery**
## Proposed change

Improve documentation of `homeworks.send_command` to show how to send a sequence of commands, including a delay

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/114059
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/homeworks.markdown [hunk 1]
```diff
@@ -37,4 +37,18 @@ Send a custom command to the Lutron Homeworks controller.
...
+#### Sending a list of commands 
+
+The example shows how to send `KBP`, wait 0.5 seconds, then send `KBR` to simulate a keypad button keypress with a duration of a half second.
+
+```yaml
+service: homeworks.send_command
+data:
+  controller_id: "homeworks"
+  command:
+    - "KBP, [02:08:02:01], 1"
```
Comment (MartinHjelmare) [inline source/_integrations/homeworks.markdown:52]: What does this mean? What's the command syntax? Where can the user find what commands are available?

**Review by MartinHjelmare — CHANGES_REQUESTED** [review]

File: source/_integrations/homeworks.markdown (hunk 1 above)
Comment (emontnemery) [reply source/_integrations/homeworks.markdown:52]: The command set is documented here: https://assets.lutron.com/a/documents/hwi%20rs232%20protocol.pdf, should we link to it?

Comment (MartinHjelmare) [reply source/_integrations/homeworks.markdown:52]: Yes.

**Review by frenck — APPROVED** [review]
Thanks, @emontnemery 👍 

../Frenck

**Pull request merged**